### PR TITLE
Fixed broken link on documentation

### LINKED ISSUE #14234: Example link points to localhost:4000
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
image

### What changes are you suggesting?

The "push event" hyperlink points to `http://localhost:4000/en/actions/learn-github-actions/events-that-trigger-workflows#push`

This should point to the push event documentation `https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push`

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by camargomarin**
This commit fixes a broken link for the 'push' event hyperlink

### Why:

Closes #14234

### What's being changed:

As specified on #14234, there is a broken link for the [documentation regarding push events](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) that points to a `localhost:4000` page, this PR modifies that link so it points to the actual documentation

Thank you!

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by ramyaparimi — DISMISSED** [review]
@Mr-Camargo 
Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

**Comment by Mr-Camargo** [comment]
> @Mr-Camargo 
> 
> Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

Hello again! Looks like a check isn't passing, how can I fix it? I don't think I did anything that would affect the run of that check.

Thank you!

File: content/actions/learn-github-actions/contexts.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ The `github` context contains information about the workflow run and the event t
...
+| `github.event` | `object` | The full event webhook payload. You can access individual properties of the event using this context. This object is identical to the webhook payload of the event that triggered the workflow run, and is different for each event. The webhooks for each {% data variables.product.prodname_actions %} event is linked in "[Events that trigger workflows](/articles/events-that-trigger-workflows/)." For example, for a workflow run triggered by the [`push` event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push), this object contains the contents of the [push webhook payload](/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push). |
```
Comment (lucascosti) [inline content/actions/learn-github-actions/contexts.md:190]:
```suggestion
| `github.event` | `object` | The full event webhook payload. You can access individual properties of the event using this context. This object is identical to the webhook payload of the event that triggered the workflow run, and is different for each event. The webhooks for each {% data variables.product.prodname_actions %} event is linked in "[Events that trigger workflows](/articles/events-that-trigger-workflows/)." For example, for a workflow run triggered by the [`push` event](/actions/using-workflows/events-that-trigger-workflows#push), this object contains the contents of the [push webhook payload](/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push). |
```

**Review by lucascosti — APPROVED** [review]
Thanks for noticing this, @Mr-Camargo! I just pushed a tiny fix to the change, and I'll merge this as soon as tests pass 👍

**Comment by Mr-Camargo** [comment]
> Thanks for noticing this, @Mr-Camargo! I just pushed a tiny fix to the change, and I'll merge this as soon as tests pass 👍 

Thank you for your help @lucascosti! 😁

**Pull request merged**